src/components/elements/Inputs/Dropdown/Dropdown.tsx

```diff
@@ -135,14 +135,31 @@ const Dropdown = (props: PropsWithChildren<DropdownProps>) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [otherIsSelected, otherValue, t]);
 
+  const getColorStatus = (option: string): string => {
+    const colorMap: { [key: string]: string } = {
+      approved: "bg-green",
+      submitted: "bg-blue",
+      draft: "bg-pinkCustom",
+      "Under Review": "bg-tertiary-600",
+      "Needs More Info": "bg-tertiary-600"
+    };
+
+    return colorMap[option] || "";
```

Comment:
Please use `??` for assigning default values.

Reply:
done